back/multiplayer.py

```diff
@@ -1,39 +1,27 @@
 import random
 
-cards = [ "Schell 7",
-          "Schell 8",
-          "Schell 9",
-          "Schell 10",
-          "Schell Unter",
-          "Schell Ober",
-          "Schell König",
-          "Schell Ass",
-          "Herz 7",
-          "Herz 8",
-          "Herz 9",
-          "Herz 10",
-          "Herz Unter",
-          "Herz Ober",
-          "Herz König",
-          "Herz Ass",
-          "Blatt 7",
-          "Blatt 8",
-          "Blatt 9",
-          "Blatt 10",
-          "Blatt Unter",
-          "Blatt Ober",
-          "Blatt König",
-          "Blatt Ass",
-          "Eichel 7",
-          "Eichel 8",
-          "Eichel 9",
-          "Eichel 10",
-          "Eichel Unter",
-          "Eichel Ober",
-          "Eichel König",
-          "Eichel Ass" ]
+colors = [
+    "Schell",
+    "Herz",
+    "Blatt",
+    "Eichel"
+]
+
+values = [
+    "7",
+    "8",
+    "9",
+    "10",
+    "Unter",
+    "Ober",
+    "König",
+    "Ass"
+]
+
 
 def get_hand(player, seed=1337):
     random.seed(seed)
-    current_game_cards = random.sample(cards, k=len(cards))
-    return current_game_cards[player*10:player*10+10]
+    current_game_cards = random.sample(range(32), k=32)
+    player_cards = current_game_cards[player * 10:player * 10 + 10]
+    player_cards.sort()
+    return [f"{colors[card // 8]} {values[card % 8]}" for card in player_cards]
```

Comment:
This builds three lists and a slice while the old code only built one list and a slice. Usage of sorted rather than list.sort is preferred.
Also the use of magic numbers became exorbitantly high, in addition to the 10s we also have 32s and 8s know.
Especially in case of the 8s some brain power has to be invested by the reader.
I suggest rewriting these lines  so that they are like the old ones, where the return value is wrapped in `sorted()`.
The lists `colors` and `values` can be elegantly combined to something like the old `cards` using a list comprehension and `itertools.product`.